cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1113,7 +1113,7 @@ declare namespace Deno {
   /** see: https://w3c.github.io/permissions/#permissionstatus */
   export class PermissionStatus {
     state: PermissionState;
-    constructor(state: PermissionState);
+    constructor();
```

Comment:
You could do a `private constructor();` I believe to help give type guidance that this cannot be constructed.

Reply:
This is to match `lib.dom.d.ts` (besides preserving the use of the class syntax).

Reply:
fair enough...